File: 0001 Code Chef/0017 Healthy Sleep/implementation/0004 java/HealthySleep.java
import java.util.Scanner;

public class HealthySleep{
    public static void main(String[] args){
        Scanner console = new Scanner(System.in);
        String stringInput = console.nextLine();
        int sleep = Integer.parseInt(stringInput);
        if(sleep == 8){
            System.out.println("PERFECT");
        }else if(sleep < 8){
            System.out.println("LESS");
        }else if(sleep > 8){
            System.out.println("MORE");
        }else{
            System.out.println("Invalid input");
        }
    }
}

File: 0001 Code Chef/0017 Healthy Sleep/implementation/0001 c/healthy_sleep.c
#include <stdio.h>

int main(){
    int sleep;
    scanf("%d",&sleep);
    if(sleep < 8){
        printf("LESS\n");
    }else{
        if(sleep > 8){
            printf("MORE\n");
        }else{
            printf("PERFECT\n");
        }
    }
    return 0;
}